readme.md

```diff
@@ -158,6 +158,28 @@ You can use the [Angular.js binding](https://github.com/hrajchert/angular-screen
 </div>
 ```
 
+#### Toggle Fullscren as an Angular 2 Directive
```

Comment:
The title should be consistent with the Angular 1 example title. Either change the other one or this.